common/src/main/java/org/cloudfoundry/identity/uaa/web/CorsFilter.java

```diff
@@ -76,6 +76,9 @@
 
     private final List<Pattern> corsXhrAllowedOriginPatterns = new ArrayList<Pattern>();
 
+    @Value("#{'${cors.xhr.allowed.headers:Accept,Authorization}'.split(',')}")
```

Comment:
The preference is still to keep the configuration in the XML files, and not let it trickle into the Java files